docs/import/manage_disaster.js

```diff
@@ -770,8 +778,24 @@ function initializeScoreSelectors(states) {
  * @param {Array<string>} assets List of assets in the disaster folder
  */
 function initializeDamageSelector(assets) {
-  createAssetDropdown(
-      assets, ['damage_asset_path'], $('#damage-asset-select').empty());
+  const mapBoundsDiv = $('#map-bounds-div');
+  const damagePropertyPath = ['damage_asset_path'];
+  const select = createAssetDropdown(
+      assets, damagePropertyPath, $('#damage-asset-select').empty());
+  select.on('change', (event) => {
+    const val = $(event.target).val();
+    hasValue(val) ? mapBoundsDiv.hide() : mapBoundsDiv.show();
+    handleScoreAssetSelection(val, damagePropertyPath);
+  });
+  const swPath = ['map_bounds_sw'];
+  const nePath = ['map_bounds_ne'];
+  const swInput = $('#map-bounds-sw');
+  swInput.val(getElementFromPath(swPath));
+  addChangeHandler(swInput, swPath);
+  const neInput = $('#map-bounds-ne');
```

Comment:
Is it weird that if you input one and not the other coord it gets stored?

Reply:
Don't think so? You have to find them one at a time (I'm hoping someone can write documentation about how to do this if we don't have it already (https://support.google.com/maps/answer/18539?co=GENIE.Platform%3DDesktop&hl=en) and link it here? @ruthtalbot ), so there's going to be a delay in inputting them. If only one is input, we won't allow processing.